specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/workspaceRP.json

```diff
@@ -3629,6 +3708,13 @@
       "x-ms-parameter-location": "method",
       "pattern": "^[a-zA-Z0-9][a-zA-Z0-9_-]{2,32}$"
     },
+    "AOAIModelsToDeployParameter": {
+      "in": "query",
+      "name": "aoaiModelsToDeploy",
+      "description": "query parameter for which AOAI mode should be deployed",
+      "type": "string",
```

Comment:
>   "type": "string", [](http://example.com/codeflow?start=3&length=20)

AOAI modes? Sounds a bit like an enum...
what are valid values?
is there actually an enum of allowed values you could declare this to be? (Maybe a reference to one you already defined somewhere.)
Or can it at least be clarified in the description?

Reply:
it will be a string with actual comma separated modes, e.g. gpt-35-turbo,gpt-4,text-ada-001